### PR TITLE
Fixed importing FieldDoesNotExist exception class for Django==3.1

### PR DESCRIPTION
**Pull request by DrMartiner**

### CODE REVIEW

**Comment by uhurusurfa** [comment]
@paulocheque - adding to the requests above. Great app you have built and would be good to get this merged.

**Review by jayvdb — APPROVED** [review]

**Review by pkasemir — APPROVED** [review]
Please merge this, it would be great to use with Django 3.1+

**Comment by pkasemir** [comment]
@paulocheque Can you merge this?

**Comment by clearyusc** [comment]
Please merge!

**Comment by s-i-l-k-e** [comment]
@paulocheque how is this progressing?

**Comment by uhurusurfa** [comment]
Anyone interested in being a member  on a fork of this project until such time as Paulo decides to allow more developers to help him so that this type of scenario does not happen again?
That way we can provide timely solutions as  Django evolves and not stall other projects that use this one.
If and when Paulo decides to actively develop this app with more (trusted) people allowed to review and merge code fixes thus eliminating the scenario we find ourselves in then we can merge back in?

**Comment by RaddishIoW** [comment]
I'm up for helping on the fork. This is preventing me from updating to 3.1 - the only thing preventing me from updating.

**Comment by RaddishIoW** [comment]
@paulocheque committed a change to the Readme 10 days ago, so he's definitely still active on the project. Just a shame he hasn't approved this pull request yet. He might not have seen it I guess, but maybe he needs some other people approved to help out on GitHub?

**Comment by jheld** [comment]
While at it, can you open an issue for PyPI ownership transfer? It may take some time, but since he has been uncommunicative despite some recent activity, having a fork and PyPI will make this seamless for users long-term.

@paulocheque We understand, life gets busy, all sorts of things happen (and we appreciate the work you have done); but making changes while ignoring critical updates provided by the community, do leave some folks with frustration.

**Comment by iamjstates** [comment]
Merging this would be greatly appreciated

File: django_dynamic_fixture/django_helper.py [hunk 1]
```diff
@@ -7,11 +7,15 @@
...
 import django
 from django.apps import apps
-from django.db import models
+from django.db import models  # noqa
 from django.db.models import *
-from django.db.models.fields import NOT_PROVIDED, AutoField, FieldDoesNotExist
+from django.db.models.fields import NOT_PROVIDED, AutoField
 from django.db.models.base import ModelBase
 from django.db.models.query import QuerySet
+try:
```
Comment (JanMalte) [inline django_dynamic_fixture/django_helper.py:15]: I would suggest to remove the try/except block and only import from `django.core.exceptions`, as the exception was moved there 6 years ago. Only the legacy reference in `django.db.models.fields` was widely used and is removed with the 3.1 release.

**Review by JanMalte — CHANGES_REQUESTED** [review]

**Comment by vecchp** [comment]
Hi, this issue is also showing up as a blocker for https://github.com/saxix/django-adminactions/pull/173.  Any way we can move this forward?

**Comment by jheld** [comment]
You can open up an issue for PyPI ownership.

**Comment by paulocheque** [comment]
Hello guys, I am sorry for the delay, I will puy you guys as collaborators of the project ok?

**Comment by paulocheque** [comment]
Version 3.1.1. released with this PR!

**Comment by RaddishIoW** [comment]
@paulocheque Thank you for this - I (and I'm sure everyone else) appreciate the work you've done on this project. Thanks for making me a collaborator. I'll help out as I can, even if it's just keeping an eye out on GitHub for things like this.

**Pull request merged**